core/src/test/scala/precog/AutoBumpSpec.scala

```diff
@@ -81,7 +92,7 @@ class AutoBumpSpec
       result must beSome(matchA[PullRequestDraft].title("Auto Bump 1"))
     }
 
-    "ignore pull requests not to master" in {
+    "ignore pull requests not to master or main - master" in {
```

Comment:
Shouldn't this be `ignore pull requests not to master or main`?

Reply:
I added `- master` because it's checking that it does find `master` here. In the same way, l.118 checks that it finds `main`. 

Or do you mean something different?